app/src/androidTest/java/com/epfl/drawyourpath/community/TournamentCreationFragmentTest.kt

```diff
@@ -215,9 +195,9 @@ class TournamentCreationFragmentTest {
         Intents.release()
     }
 
-    @Test
+    /*@Test

```

Comment:
Why this test is commented, maybe correct it or remove it if it not used.

Reply:
I will remove them